packages/api/src/routes/medical/__tests__/mapi/e2e/mapi.test.e2e.ts

```diff
@@ -0,0 +1,173 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { Organization, Facility, Patient } from "@metriport/api-sdk";
+import { Organization as CWOrganization } from "@metriport/commonwell-sdk";
+import { Organization as FhirOrg } from "@medplum/fhirtypes";
+import { faker } from "@faker-js/faker";
+import cwCommands from "../../../../../external/commonwell/__tests__";
+import { validateCWOrg, validateFhirOrg, validateLocalOrg } from "./organization";
+import { createFacility, validateFacility } from "./facility";
+import { createPatient, validateFhirPatient, validateLocalPatient } from "./patient";
+import { createConsolidated } from "./consolidated";
+import { retryFunction, fhirHeaders, ResourceType, fhirApi, medicalApi } from "./shared";
+import { Util } from "../../../../../shared/util";
+import { Config } from "../../../../../shared/config";
+
+const maxRetries = 4;
+
+jest.setTimeout(30000);
+
+// NEVER TO BE RUN IN PRODUCTION
+if (Config.isStaging() || !Config.isCloudEnv()) {
+  describe("MAPI E2E Tests", () => {
+    let facility: Facility;
+    let patient: Patient;
+
+    it("gets an organization", async () => {
+      const org = await medicalApi.getOrganization();
+
+      if (!org) {
+        throw new Error("Org not found");
+      }
```

Comment:
Ideally we would not throw errors on tests, could do something like this in this situation:

```ts
expect(org).toBeTruthy()
```

Here and all other places w/ exceptions being thrown from tests